pom.xml

```diff
@@ -32,14 +32,16 @@
         <thymeleaf_version>3.1.2.RELEASE</thymeleaf_version>
         <cqframework_version>1.4.9</cqframework_version>
     </properties>
+    <groupId>com.accurids.fhir</groupId>
 
-    <artifactId>org.hl7.fhir.core</artifactId>
+    <artifactId>core</artifactId>
     <packaging>pom</packaging>
 
-    <name>HL7 Core Artifacts</name>
+    <name>ACCURIDS Secured FHIR Core</name>
+    <description>FHIR Core 5.1.0 with CVE patches</description>
```

Comment:
Maybe write it more generic like

`<description>Fork of HL7 FHIR Core with applied CVE-related security patches</description>`

as we may not only release 5.1.0 patched version, it could be other version too in the future.